connexion/api.py

```diff
@@ -181,10 +183,8 @@ def add_paths(self, paths=None):
                     if self.debug:
                         logger.exception(error_msg)
                     else:
-                        import sys
                         logger.error(error_msg)
-                        et, ei, tb = sys.exc_info()
-                        raise ei.with_traceback(tb)
+                        six.reraise(*sys.exc_info())
```

Comment:
unrelated to this PR, but I hit this the other day on python2, so figured I'd fix it